docs/rooms/fireRoom.md

```diff
@@ -0,0 +1,59 @@
+# Fire Room
+
+A room with three different paths.
+One path leads to a person, the second path leads to a item and the third path leads no where.
+Each path is on fire but the player is given two buckets of water.
+The goal of this room is to save the person.
+
+## PlayerStates
+
+- **AtEntrance**: The player is in the main room with x buckets of water.
+- **DouseForward**: The water from the bucket puts out the fire in front of the player.
+- **DouseLeft**: The water from the bucket puts out the fire to the left of the player.
+- **DouseRight**: The water from the bucket puts out the fire to the right of the player.
+- **Nothing**: There is no water left. You did nothing.
+- **SavedPerson**: Congratulations!!! The player saved the person!!!
+- **ItemRetrieved**: Congratulations!!! The player got a cool item!!!
+- **EmptyPath**: There is nothing here. Better luck next time.
+- **Flameless**: You already put out the fire here...
```

Comment:
"PlayerState" was an oversimplification to help us design some starting rooms. Now that we have more data in the room state, we should find a way to write our diagrams to reflect the entire room state.